### PR TITLE
Add Paragrapgh. #2

### PR DESCRIPTION
**Pull request by sadelcode**
I'll work on it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,6 @@
     </head>
     <body>
         <h1 style="color:red">Welcome to Potionfy!</h1>
+        <p>This is a new paragrapgh</p>
```
Comment (RashedCode) [inline index.html]: Please make it bigger

**Review by RashedCode — CHANGES_REQUESTED** [review]
make a change

File: index.html (hunk 1 above)
Comment (sadelcode) [reply index.html]: Sure I'll do that

**Review by RashedCode — APPROVED** [review]
Well done

**Pull request merged**